common/src/View/FaceAttribsEditor.cpp

```diff
@@ -158,7 +158,7 @@ namespace TrenchBroom {
             }
 
             Model::ChangeBrushFaceAttributesRequest request;
-            if (setFlag) {
+            if (setFlag & 1 << index) {
```

Comment:
use parens around (1<<index) for clarity

Reply:
agreed